### PR TITLE
#4199 text inputs

### PR DESCRIPTION
**Pull request by desfero**
**Notes**
- Password and suffixes are not implemented as they are not a part of MVP
- The android build was fixed

### CODE REVIEW

File: packages/wallet/app/components/shared/forms/layouts/Field.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+storiesOf("Molecules|Field", module)
+  .add("default input", () => (
+    <Field
+      label="User name"
+      type={EFieldType.INPUT}
+      placeholder="Lorem ipsum..."
+      helperText="It's an important information for us to provide the best user experience"
+    />
+  ))
+  .add("default input without labeel", () => (
```
Comment (desfero) [inline packages/wallet/app/components/shared/forms/layouts/Field.stories.tsx]:
```suggestion
  .add("default input without label", () => (
```

File: packages/wallet/app/components/shared/forms/layouts/TextInput.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  editable,
+  invalid,
+  ...props
+}) => {
+  const [hasFocus, setHasFocus] = React.useState(false);
+
+  const handleFocus = (event: NativeSyntheticEvent<TextInputFocusEventData>) => {
+    setHasFocus(true);
+
+    onFocus?.(event);
```
Comment (desfero) [inline packages/wallet/app/components/shared/forms/layouts/TextInput.tsx]: what do you think, which version is not readable
```
if (onFocus) {
    onFofus(event);
}
```

File: packages/wallet/app/components/shared/forms/layouts/Field.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      return <TextAreaInput {...rest} />;
+    default:
+      assertNever(type, "Invalid input type");
+  }
+};
+
+/**
+ * A wrapper around label, input, help text and error message to just provide single top level api to be used in components
+ */
+const Field: React.FunctionComponent<TExternalProps> = ({
```
Comment (arjunu) [inline packages/wallet/app/components/shared/forms/layouts/Field.tsx]: can we call this TextField?

File: packages/wallet/app/components/shared/forms/layouts/TextInput.tsx (hunk 2 above)
Comment (arjunu) [reply packages/wallet/app/components/shared/forms/layouts/TextInput.tsx]: I vote for Elvis

File: packages/wallet/app/components/shared/forms/layouts/TextInput.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { View } from "react-native";
+
+import { LineBreak } from "../../LineBreak";
+import { TextInput } from "./TextInput";
+
+storiesOf("Atoms|TextInput", module).add("default", () => (
```
Comment (arjunu) [inline packages/wallet/app/components/shared/forms/layouts/TextInput.stories.tsx]: can you add one story with `editable = false`

**Pull request merged**